### ISSUE TITLE
Options page

### ISSUE THREAD

**Issue by JettBurns14** [ka-extension/ka-extension-ts#128]
This options page would allow users to enable or disable certain features. Examples of usage could include disabling the extra wide Editor, viewing program info, some CSS changes (like for buttons), showing Guidelines or not, and other stuff.

This guide explains pretty much exactly what I'm imagining:
https://developer.chrome.com/extensions/options

**Comment by JettBurns14** [comment]
BTW my idea of design would use sliders like this:
![image](https://user-images.githubusercontent.com/24257648/48927307-7465c100-ee9a-11e8-804a-7c3e03bf6ca0.png)

**Comment by An0n3m0us** [comment]
I personally think an options tab would be preferred.
E.g:
![image](https://user-images.githubusercontent.com/49789044/109879120-132e1a00-7c6d-11eb-8cd2-90c8acabee33.png)

The reason being that going to the extension page, searching for the extension and then finally getting to the options page is a little slower and tedious.

**Comment by JettBurns14** [comment]
Agreed! I think the whole popup design should be redone TBH, to better accommodate this possibility and future additions to the popup... I'd have to look for ideas, but the current design feels very limited.
